pom.xml

```diff
@@ -434,7 +443,7 @@
               <properties>
                 <property>
                   <name>log4j.configurationFile</name>
-                  <value>${project.baseUri}src/main/resources/log4j2-test.xml</value>
+                  <value>${project.baseUri}src/main/resources/log4j2.xml</value>
```

Comment:
Was this the cause of the log4j issue?

Reply:
This is the cause of an error or perhaps a warning message being displayed.
I would say that this is part of the problem, but my local build still failed even with this change.